src/utils/filters/time-entry-filter/timeEntryFilterBuilder.js

```diff
@@ -6,26 +6,41 @@ import {
   joinAndConditions,
 } from '../resource-filter-builder/resourceFilterBuilder';
 
-const filterTimeEntriesOnDate = (resourceDateProperty, date) => {
-  const startOfDateFilter = filterOnOrAfterDate(resourceDateProperty, date);
+const filterStartTimeEntriesOnDate = (resourceDateProperty, date) => {
+  const startOfDateFilter = filterOnOrAfterDate(
+    resourceDateProperty,
+    dayjs(date).subtract(1, 'day')
```

Comment:
Is it worth pulling out the day offset here into a constant purely so that it has a variable name associated with it so that we know in the future what `1` is for? Same for line 28.
Or maybe add some JsDoc instead. 
Just thinking we may return to this 6 months later and wonder what the significance was for adding 1 or 2 days at either end of the filter.

Reply:
I've added some constants for this